packages/secure_storage/amplify_secure_storage/Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: pigeons
```

Comment:
This PR depends on https://github.com/aws-amplify/amplify-flutter/pull/1644. See https://github.com/aws-amplify/amplify-flutter/pull/1657/files/bf1631f054f95598187f6355091be8352efe7726..9fa83ab266e98454ca5280f3ecadcb3f3589d939 for the changes that are new in this PR